File: CodeWars/Java/8Kyu/NthPower.java
// URL: https://www.codewars.com/kata/57d814e4950d8489720008db/train/java

public class NthPower {
  public static int nthPower(int[] array, int n) {
    int number = (array.length > n) ? (int) Math.pow(array[n], n) : -1;
    return number;
  }
}

File: CodeWars/JavaScript/8Kyu/N-thPower.js
// URL: https://www.codewars.com/kata/57d814e4950d8489720008db/train/javascript

function index(array, n){
  return array.length > n ? Math.pow(array[n],n) : -1;
}

console.log(index([1, 2, 3, 4],2))